<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>663</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botao_escrever">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Escrever dados</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="logo_farmacia">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>192</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="logo_pet">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_ler">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ler dados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_editar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar dados</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_ler">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>881</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_editar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_escrever">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>881</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
background-color: rgb(255, 0, 127);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
